bundles/org.openhab.binding.salus/src/main/resources/OH-INF/thing/it600.xml

```diff
@@ -79,5 +91,9 @@
 		<item-type>Switch</item-type>
 		<label>Running State</label>
 		<description>Is the device running</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Power</tag>
```

Comment:
I'm not sure about those tags. This channel shows if the (heating) pump works at the time. It cannot be changed by user, it's controlled by internal algorithm of the device. Also I do not know what `power` tag means in this context